src/styles/day.scss

```diff
@@ -0,0 +1,32 @@
+@import './variables.scss';
+
+.calendar__day {
+  width: $day-size;
+  height: $day-size;
+  background-color: #eee;
+  border: 1px solid #000;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: $font-family;
```

Comment:
Ensure that `$font-family` is defined in your `variables.scss` file. Undefined variables will cause compilation errors.